_pages/en/strategy-technology-management.md

```diff
@@ -0,0 +1,338 @@
+
+<p>Authorities to establish such instrument at ESDC is at CIO and CFO level.</p>
+    </td>
+  </tr>
+  <tr>
+    <td class="action-header"><strong>3. Implement DevSecOps</strong></td>
+    <td rowspan="2">(removed)</td>
+  </tr>
+  <tr>
+    <td>
+<p>Inspired by the <a href="https://software.af.mil/team/platformone/">US Air Force's Platform One team</a>, DevSecOps focuses on SecOps (security at runtime) and <i>incentivizes</i> re-using existing software components and applications instead of proliferating new ones (the #1 help for IT Security).</p>
+
+<p>Architecture and release management require to adapt to realize the benefits to DevSecOps.</p>
+
+<p>Architecture establishes managed services for software factories so development teams can focus on building and deploying secure Program applications with ease in conformance with <a href="https://esdc-iitb-sdlc-pdg-staging.azurewebsites.net/">ESDC's SDLC</a>.</p>
+
+<p>Release Management establishes clear guardrails, coded in software delivery pipelines, and actively favours automation to assess the readiness of software to reach production state.</p>
+
+<p>Teams (software factories) actively monitor their production software for both user feedback and anomalies to quickly remediate them.</p>
+
+<p>IT Security makes use of advance tools to not only monitor in production but automatically respond to threats.</p>
+
+<p>DevSecOps is more than pipelines, it ensures IT maintains pace of relevance with Program needs.</p>
+    </td>
+  </tr>
+  <tr>
+    <td class="action-header"><strong>4. Establish Technology Procurement Mandatory Procedures</strong></td>
+    <td rowspan="2">(removed)</td>
+  </tr>
+  <tr>
+    <td>
+<p><i>Adopting</i> commercially adopted open-source software and <a href="https://derisking-guide.18f.gov/state-field-guide/budgeting-tech/#procure-services-not-software">leveraging industry services</a> to help ESDC integrate and adapt them to Program needs must be seriously evaluated before going to market.</p>
+
+<p>Procuring commercial software is one option out of many.
+Adopting commercially adopted open-source software to help ESDC in <a href="https://sara-sabr.github.io/ITStrategy/open-first-strategy.html">ensuring its technology sovereignty</a> and comply with the <a href="https://www.tbs-sct.canada.ca/pol/doc-eng.aspx?id=32601">Directive on Service and Digital</a> requirement 4.4.3.12. ESDC already has an <a href="https://014gc.sharepoint.com/:w:/r/sites/CO7397744/MasterLib/ESDC%20Open%20Source%20Management%20Framework%20v%201.2.docx?d=w71619962ea18494a9d8fbd9e0bcecff7&csf=1&web=1&e=9lngyn">Open Source Software (OSS) Management Framework</a> that sets direction and guidance around OSS product selection considerations and compliance.</p>

```

Comment:
Could this be "Adopting commercially available open-source software"?  But also, is it right to use commercially here? Does commercially imply a procurement?  Could we just say "Adopting open-source software..."?

Reply:
Ditto above. we must be using this phrase for a reason, I'd be curious why...are we saying we should only adopt open source software that has been well tested by industry?